advanced_reflector_detector_node:
  ros__parameters:
    lidar_topic: "/lidar_points"
    yolo_topic: "/yolo_result"
    camera_info_topic: "/camera_info"
    pose_topic: "/localization/pose"
    roi_cloud_topic: "/roi_pointcloud"
    reflector_poses_topic: "/reflector_poses"
    reflector_markers_topic: "/reflector_markers"
    projected_cloud_topic: "/projected_pointcloud"
    volume_cloud_topic: "/volume_pointcloud"
    reflector_class_id: "reflector"
    volume_height: 0.5
    ndt_score_threshold: 0.8